### ISSUE TITLE
v3.3.2+: Cannot find module

### ISSUE THREAD

**Issue by josephzidell** [kucherenko/jscpd#393]
**Describe the bug**
v3.3.1 works fine. Upgraded to 3.3.2, and it gives this error:
```
Cannot find module '/Users/<xyz>/<project>/src/frontend/node_modules/jscpd/bin/../dist'
Require stack:
- /Users/<xyz>/<project>/src/frontend/node_modules/jscpd/bin/jscpd
```

**To Reproduce**
1. Set `"jscpd": "3.3.2"` in `package.json` and run `npm install`
2. Run `npx jscpd src`
3. See error

**Expected behavior**
It runs and shows number of duplications

**Desktop (please complete the following information):**
 - OS: Mac
 - OS Version Catalina 10.15.5
 - NodeJS Version v13.8.0
 - jscpd version 3.3.2

**Comment by kucherenko** [comment]
During publication the jscpd@3.3.2 I had error and that is why I published new version. 
I'm going to disable version 3.3.2.
I suggest to use jscpd@3.3.9.

**Comment by josephzidell** [comment]
Thanks @kucherenko! I upgraded to 3.3.9, and am getting the same error

**Comment by kucherenko** [comment]
Oh... will fix, thank you

**Comment by kucherenko** [comment]
Working on it

**Comment by kucherenko** [comment]
Sorry for that - it was broken build process

fixed at jscpd@3.3.13

**Comment by josephzidell** [comment]
Thanks so much @kucherenko! Works now.
